docs/profiler.md

```diff
@@ -0,0 +1,112 @@
+# What is the profiler tool
+
+The major goal of profiler tool is to provide an estimation how fast can be the proof created.
+Creating a proof in spacemesh depends on few major factors:
+* CPU speed
+* Disk speed
+* Amount of storage initialized
+
+The major task of the profiler tool is to estimate how fast can be the proof created for a given CPU speed and disk speed.
```

Comment:
Actually, IMO there are 2 tasks:
- estimate how much storage one can safely initialize (to be able to generate POST proof later)
- configure proving (nonces and threads) optimally (to best use resources, so that disk and CPU are in balance)

Reply:
True will adjust.

Reply:
Done